pennylane_lightning_gpu/src/simulator/StateVectorCudaMPI.hpp

```diff
@@ -0,0 +1,1454 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
```

Comment:
Just want to confirm if it is proper to put 2018-2023 here instead of 2022-2023 here since L-GPU are released last year and the API involved here was release two month ago. @mlxd

Reply:
Both ways are good for me. We just need to be consistent and cover the whole (proper) time.

Reply:
Have updated all the files that are created or modified in this PR. Other changes will be made in another PR.